resdk/resources/permissions.py

```diff
@@ -269,3 +276,43 @@ def __repr__(self):
                 ))
 
         return '\n'.join(res)
+
+    def _get_holders_with_perm(self, perm):
+        """Get Users/Group/public names that have ``perm`` perm."""
+        # Prevent circular imports
+        from .user import Group, User
+
+        self.fetch()
+
+        holders = []
+        for item in self._permissions:  # pylint: disable=protected-access
+            if perm in item['permissions']:
+                if item['type'] == 'user':
+                    holders.append(
+                        User(
+                            self.resolwe,
+                            id=item['id'],
+                            first_name=item['name'],
+                            username=item['username'],
+                        ),
+                    )
+                elif item['type'] == 'group':
+                    holders.append(Group(self.resolwe, id=item['id'], name=item['name']))
+                else:
```

Comment:
Change to `elif item['type'] == 'public'` for a sanity check.

Reply:
:+1: